timeforplanb123/nornir_cli
# Update dependencies and commands, import reorder, bump version to 1.1.0

- Update dependencies to latest versions, add new dev-dependencies:
  - see `pyproject.toml`
- Add new custom `Nornir` runbook examples. This is an easy way to add your `Nornir` runbooks to the `nornir_cli` interface
- Fix `short_help` and `--help` command options parsers
- Import reorder with `flake8-import-order`
- Update docs:
  - fix color palette toggle configuration (`mkdocs.yml`)
  - update docs according to the new dependencies versions
- Bump version to `1.1.0`
